templates/zerver/help/unread-counter-and-pointer.md

```diff
@@ -0,0 +1,44 @@
+# How the unread counter and navigation work
+
+## Unread messages
```

Comment:
This section feels a little small.  Other things that we could add
* Discussion of where unread counts will be displayed
* Description of the visual cue for an unread message
and maybe
* Discussion of bankruptcy